v2/slogger/rolling_file_appender/state.go

```diff
@@ -0,0 +1,64 @@
+package rolling_file_appender
+
+import (
+	"encoding/json"
+	"os"
+	"path/filepath"
+	"time"
+)
+
+type state struct {
+	LogStartTime time.Time
+}
+
+func newState(logStartTime time.Time) *state {
+	return &state{logStartTime}
+}
+
+func readState(path string) (*state, error) {
```

Comment:
Good catch.  There is not.  None of the tests create an appender for which a state file already exists.  I will add such a test